src/System.Security.SecureString/tests/SecureStringTests.cs

```diff
@@ -444,6 +445,64 @@ public static void Grow_Large()
             }
         }
 
+        [OuterLoop]
+        [Theory]
+        [InlineData(5)]
+        public static void ThreadSafe_Stress(int executionTimeSeconds) // do some minimal verification that an instance can be used concurrently
+        {
+            using (var ss = new SecureString())
+            {
+                DateTimeOffset end = DateTimeOffset.UtcNow + TimeSpan.FromSeconds(executionTimeSeconds);
+                Task.WaitAll(Enumerable.Range(0, Environment.ProcessorCount).Select(_ => Task.Run(() =>
+                {
+                    var rand = new Random(Task.CurrentId.Value);
```

Comment:
Though I guess the problem is theoretically that it was a concurrency error... so perhaps not useful.